Tasks/NuGetCommandV2/nugetcustom.ts

```diff
@@ -35,12 +35,12 @@ export async function run(nuGetPath: string): Promise<void> {
     }
 
     try {
-        let credProviderPath = nutil.locateCredentialProvider();
-
         // Clauses ordered in this way to avoid short-circuit evaluation, so the debug info printed by the functions
         // is unconditionally displayed
         const quirks = await ngToolRunner.getNuGetQuirksAsync(nuGetPath);
-        const useCredProvider = ngToolRunner.isCredentialProviderEnabled(quirks) && credProviderPath;
+        const useCredProvider: boolean = ngToolRunner.isCredentialProviderEnabled(quirks);
+        const useV2CredProvider: boolean = ngToolRunner.isCredentialProviderV2Enabled(quirks) && useCredProvider === true;
```

Comment:
Don't check for `&& useCredProvider === true` as they're independent of each other. We wouldn't want to disable V2 if V1 is disabled.